docs/DeveloperGuide.md

```diff
@@ -297,14 +297,53 @@ The activity diagram shows the general sequence of steps when a user interacts w
 ### Find appointment feature
 
 #### Implementation
+When a user types a `find appt` command, the DocTrack application will 
+find the appointment based on the parameters provided.
+
+**Step 1**. The user types the `find appt` command in the `CommandBox` followed by the name or a date
+The date is specified using a `d/` prefix and the name is specified using `n/` prefix.
+
+**Step 2**. The command is passed to the `LogicManager`. `LogicManager` then calls the `AddressBookParser::parseCommand`
+method to parse the `find appt` command
+
+**Step 3**. The `AddressBookParser` creates a `FindAppointmentCommand` by passing the string to `FindCommandParser` 
+and is then returned to the `LogicManager`.
+
+**Step 4**. The `LogicManager` calls the `FindAppointmentCommand:execute` method which creates a new `CommandResult` Object
+
+**Step 5**. The `CommandResult` object is returned to the `LogicManager`.
```

Comment:
Can we replace all the steps with a general set of steps for the entity commands add, edit, find, delete etc? This is because the implementation steps for the entity commands are similar, so we can use a general sequence diagram and a general sequence of steps to explain the implementation for all the entity commands.

You can refer to my updated section on general commands as a reference!